### PR TITLE
Dbtp 110 codebuild project

### PR DESCRIPTION
**Pull request by ejayesh**
Manually create a custom project

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -48,8 +48,8 @@ def route53_session(aws_credentials):
...
-def iam_session(aws_credentials):
-    with mock_iam():
-        session = boto3.session.Session(profile_name="foo", region_name="eu-west-2")
-        yield session.client("iam")
\ No newline at end of file
+# @pytest.fixture(scope="function")
+# def iam_session(aws_credentials):
+#     with mock_iam():
+#         session = boto3.session.Session(profile_name="foo", region_name="eu-west-2")
+#         yield session.client("iam")
```
Comment (yusufsheiqh) [inline tests/conftest.py:55]: Do we not need this anymoar? Could be removed(?).

Comment (ejayesh) [reply tests/conftest.py:55]: yep removing

File: tests/test_dns_cli.py [hunk 2]
```diff
@@ -144,3 +144,7 @@ def test_assign_domain(check_aws_conn, check_response, ensure_cwd_is_repo_root):
     runner = CliRunner()
     result = runner.invoke(assign_domain, ["--app", "some-app", "--domain-profile", "foo", "--project-profile", "foo", "--svc", "web", "--env", "dev"])
     assert result.output.startswith("There are no clusters matching") == True
+
+
+def test_stuff(acm_session):
+    print("nothing")
```
[the remark this answers is not in the document]
Comment (ejayesh) [reply tests/test_dns_cli.py:150]: gah I thought I removed this

File: tests/conftest.py (hunk 1 above)
Comment (yusufsheiqh) [reply tests/conftest.py]: Resolved in https://github.com/uktrade/copilot-tools/pull/73/commits/45107de08e0ffe74cbfaa744aa07a5b2cd8db584.

**Review by yusufsheiqh — APPROVED** [review]

**Pull request merged**